modules/core/src/main/java/org/apache/ignite/lang/ErrorGroup.java

```diff
@@ -86,14 +87,9 @@ public int code() {
      *
      * @param errorCode Error code to be registered.
      * @return Full error code which includes group code and specific error code.
-     * @throws IllegalArgumentException If the given {@code errorCode} is already registered
-     *      or {@code errorCode} is greater than 0xFFFF or less than or equal to 0.
+     * @throws IllegalArgumentException If the given {@code errorCode} is already registered.
      */
-    public int registerErrorCode(int errorCode) {
-        if (errorCode <= 0 || errorCode > 0xFFFF) {
-            throw new IllegalArgumentException("Error code should be greater than 0 and less than or equal to 0xFFFF");
-        }
-
+    public int registerErrorCode(short errorCode) {
```

Comment:
Minor. Seems that there's no consistency in errorCode naming. Here it's errorCode, but in `public boolean isRegistered(ErrorGroup group, short code)` it's just code.

Reply:
You are right, the parameter names should be the same.
I decided to remove the `isRegistered` method just because it is not used anywhere.